CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Mon Feb 18 2019 Trevor Vaughan <tvaughan@onyxpoint.com> - 4.7.0-0
+- Update the dependency list in metadata.json
+- Fix the one_shot scenario tests
```

Comment:
were the fixes solely to fix the tests or because the one shot script was broken?

Reply:
It was just a collapse of the CHANGELOG entry since it was the same person doing the work under an unreleased version.